pages/linux/bzip2.md

```diff
@@ -0,0 +1,15 @@
+# bzip2
+
+> A block-sorting file compressor, alias - bunzip2.
+
+- Compress file:
+
+`bzip2 {{path/to/file_to_compress}}`
+
+- Decompress file:
+
+`bzip2 -d {{path/to/compressed_file.bz2}}`
+
+- Decompress to STDOUT with 'bzip2 -dc' or use an alias 'bzcat':
```

Comment:
:D I'll remove bzcat reference, thought it might be good to mention it and not create separate page for bzcat.

-c means 'Compress or decompress to standard output'.